lib/rules/no-restricted-imports.js

```diff
@@ -89,6 +89,9 @@ const arrayOfStringsOrObjectPatterns = {
                         minItems: 1,
                         uniqueItems: true
                     },
+                    regexGroup: {
+                        type: "string"
+                    },
```

Comment:
"group" in the other option's name refers to group of (gitignore) patterns. Since this is only one pattern, maybe the option name could be just `regex`?

Reply:
Yeah, makes sense to me.

Reply:
Updated.